packages/patternfly-4/react-core/src/components/LoginPage/LoginForm.js

```diff
@@ -42,9 +41,9 @@ const propTypes = {
   /** Function that handles the onChange event for the Remember Me Checkbox */
   onChangeRememberMe: PropTypes.func,
   /** Aria Label for the Remember me checkbox */
-  rememberMeAriaLabel: props => {
-    if (props.rememberMeLabel && !props.rememberMeAriaLabel) {
-      return new Error('rememberMeAriaLabel is required with the Remember me checkbox');
+  'aria-label': props => {
```

Comment:
This one might also be a separate issue, but this attribute should be removed from this checkbox. In this case, the aria-label is overwriting the text defined in the `<label>`.

Let me know if an issue is needed for this.

Reply:
Also, there's an issue in the preview. This error is displaying: 
> SyntaxError: Invalid or unexpected token

I'm not sure if this is what's causing the issue, but I see that this prop is still being defined on LoginForm even though you changed it to `aria-label`:
`rememberMeAriaLabel="Remember me Checkbox"`

Reply:
Why should the attribute be removed? This is starting to get outside the scope of this PR, so a separate issue would be nice.

Reply:
👍 I created https://github.com/patternfly/patternfly-react/issues/1737 for my first comment. 

But the second comment might still be relevant to this PR?

Reply:
The login page has been broken since #1656 [(see preview).](https://1656-pr-patternfly-react-patternfly.surge.sh/patternfly-4/components/loginpage/) Fixing it is a part of #1724 which is next on my list.